Makefile

```diff
@@ -6,8 +6,8 @@ NAME = ansible-runner
 IMAGE_NAME ?= quay.io/ansible/ansible-runner
 IMAGE_NAME_STRIPPED := $(word 1,$(subst :, ,$(IMAGE_NAME)))
 GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
-ANSIBLE_BRANCH ?= devel
-ANSIBLE_VERSIONS ?= stable-2.9 stable-2.10 devel
+ANSIBLE_BRANCH ?= ""
+ANSIBLE_VERSIONS ?= stable-2.9 stable-2.10 stable-2.11
```

Comment:
Actually, could we add back `devel` to the end of this? If we want to keep the number of versions small, we can remove 2.10. But fore now, I think it's easiest to put them all in.

Reply:
hi @AlanCoding thx for the reply,  yes of course for adding back !,  just two questions 
Do you mean to remove 2.9 (not 2.10) ?

What about this condition ? https://github.com/ansible/ansible-runner/blob/dcdb62daf668a31754fc6fbf73374e7d14d5f52c/Dockerfile#L12

if devel is an `ANSIBLE_VERSIONS` element when we execute `make image_matrix` or `make image `,
make fails due the file requirements-**devel**.txt doesn't exist

Should i create the file ?  copying the content from 2.11 ?

Reply:
yeah, as you can image, `make image_matrix` and `make image` are not being used to produce the actual images on quay. The real logic is in the Zuul configs. I wish that it were easy to reuse that same stuff for user-oriented commands, but it's not. I'm not sure what to do about the requirements. The best solution would be to emulate what Zuul does, which requires some digging.

I did mean 2.10. But I'd hold off on that. 2.9 remains important for the time being.

Reply:
Ok perfect.   I'll check what Zuul does and I'll commit the changes.